### ISSUE TITLE
Invalid event TransitionVideoEnd

### ISSUE THREAD

**Issue by ethanopp** [Elektordi/obs-websocket-py#44]
Following code is working (changing the scene in OBS)
```
        ws = obsws(host, port, password)
        ws.connect()
        ws.call(requests.SetCurrentScene('Clone Hero'))
```

But is outputting these errors in console:
```
Invalid message: {
    "duration": 300,
    "from-scene": "Clone Hero Menu Nav",
    "name": "Fade",
    "to-scene": "Clone Hero",
    "type": "fade_transition",
    "update-type": "TransitionVideoEnd"
} (Invalid event TransitionVideoEnd)
Invalid message: {
    "duration": 300,
    "name": "Fade",
    "to-scene": "Clone Hero",
    "type": "fade_transition",
    "update-type": "TransitionEnd"
} (Invalid event TransitionEnd)
```

**Comment by chepecarlos** [comment]
same problem

**Comment by S1N1C** [comment]
Hi, not sure if you are still having this issue but I think its because these two classes are not fleshed out in the events.py. 

I know its not ideal but you can prototype these two classes and just pass them in events.py and it should do away with this invalid message.

I added this to events.py
class TransitionVideoEnd(Baseevents):
    pass

class TransitionEnd(Baseevents):
    pass

A better alternative would be to implement them but this is beyond my understanding at this point.

**Comment by iarspider** [comment]
you could try this #47

**Comment by Elektordi** [comment]
Hello,

Fixed on master. Will be included in version 0.5.2 on PyPI, planned for this evening...

Best regards,
